Makefile

```
@@ -14,17 +14,13 @@ all: policy all-data
 
 policy: \
 	katello.pp.bz2 \
```

```suggestion
	katello.pp.bz2
```

Same thing needs to be done in a few places.